miso-web/src/main/webapp/WEB-INF/pages/editInstrumentModel.jsp

```diff
@@ -0,0 +1,83 @@
+<%--
+  ~ Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+
+  ~ MISO project contacts: Robert Davey @ TGAC
```

Comment:
Do we include this for historical reasons or is this inaccurate?

Reply:
Was there because copy+paste. Removed